### ISSUE TITLE
Long when clauses not readable

### ISSUE THREAD

**Issue by ramya-rao-a** [Microsoft/vscode#23491]
Testing #23287 

![image](https://cloud.githubusercontent.com/assets/16890566/24425871/89ed5d38-13ba-11e7-96d3-6d86c9241cf4.png)

Of course, you can hover to see the whole clause, but that wont work for people not using the mouse

**Comment by sandy081** [comment]
Duplicates #23378
